src/sparse/impl/KokkosSparse_spmv_bsrmatrix_impl.hpp

```diff
@@ -431,6 +431,11 @@ struct BsrMatrixSpMVTensorCoreFunctor {
       }
     }
     mbr.team_barrier();
+
+    // Suppress unused var warnings
```

Comment:
This probably works although I am a bit worried about this overall functor which relies on nvcuda::wmma and compiles on host apparently?

Reply:
I am not sure. This is a question for @cwpearson.

Reply:
Yeah, I know. We might want to add a note in a comment so that he can have a look later when he gets back.

Reply:
Is the concern that parts of the functor are guarded with `__CUDA_ARCH__` and therefore if the functor is compiled on the host it won't do anything? It's possible that's left over from when I was developing it and didn't have the various architecture guards working. @lucbv Can you open an issue and we can track it there?